test/integration/ComputeExamples.test.ts

```diff
@@ -0,0 +1,701 @@
+  providerResponse = await ProviderInstance.encrypt(ddo, providerUrl)
+  const encryptedResponse = await providerResponse
+  const validateResult = await aquarius.validate(ddo)
+  /// ```
+  /// <!--
+  assert(validateResult.valid, 'Could not validate metadata')
+  /// -->
+  /// ```Typescript
+  await nft.setMetadata(
+    nftAddress,
+    owner,
+    0,
+    providerUrl,
+    '',
+    '0x2',
+    encryptedResponse,
+    validateResult.hash
+  )
+  return ddo.id
+}
+/// ```
+
+/// Add a `handleOrder()`function.
+/// ```Typescript
+async function handleOrder(
+  order: ProviderComputeInitialize,
+  datatokenAddress: string,
+  payerAccount: string,
+  consumerAccount: string,
+  serviceIndex: number,
+  consumeMarkerFee?: ConsumeMarketFee
+) {
+  /* We do have 3 possible situations:
+     - have validOrder and no providerFees -> then order is valid, providerFees are valid, just use it in startCompute
+     - have validOrder and providerFees -> then order is valid but providerFees are not valid, we need to call reuseOrder and pay only providerFees
+     - no validOrder -> we need to call startOrder, to pay 1 DT & providerFees
+  */
+  if (order.providerFee && order.providerFee.providerFeeAmount) {
+    await approveWei(
+      web3,
+      config,
+      payerAccount,
+      order.providerFee.providerFeeToken,
+      datatokenAddress,
+      order.providerFee.providerFeeAmount
+    )
+  }
+  if (order.validOrder) {
+    if (!order.providerFee) return order.validOrder
+    const tx = await datatoken.reuseOrder(
+      datatokenAddress,
+      payerAccount,
+      order.validOrder,
+      order.providerFee
+    )
+    return tx.transactionHash
+  }
+  const tx = await datatoken.startOrder(
+    datatokenAddress,
+    payerAccount,
+    consumerAccount,
+    serviceIndex,
+    order.providerFee,
+    consumeMarkerFee
+  )
+  return tx.transactionHash
+}
+/// ```
+
+/// In your compute.ts file create a function that you can call later where you can add and test the following chunks of code
+/// <!--
+describe('Compute-to-data example tests', async () => {
+  /// -->
+
+  /// We load the configuration:
+  /// ```Typescript
+  before(async () => {
+    web3 = new Web3(process.env.NODE_URI || configHelperNetworks[1].nodeUri)
+    config = new ConfigHelper().getConfig(await web3.eth.getChainId())
+    config.providerUri = process.env.PROVIDER_URL || config.providerUri
+    addresses = JSON.parse(
+      // eslint-disable-next-line security/detect-non-literal-fs-filename
+      fs.readFileSync(
+        process.env.ADDRESS_FILE ||
+          `${homedir}/.ocean/ocean-contracts/artifacts/address.json`,
+        'utf8'
+      )
+    ).development
+    aquarius = new Aquarius(config.metadataCacheUri)
+    providerUrl = config.providerUri
+    datatoken = new Datatoken(web3)
+    /// ```
+    /// As we go along it's a good idea to console log the values so that you check they are right
```

Comment:
```suggestion
    /// As we go along it's a good idea to console log the values so that you check they are right. At the end of your `run(){ ... }` function add the following logs:
```